lib/ui/desc_persona.dart

```diff
@@ -94,11 +98,12 @@ class _DescPersonaScreenState extends State<DescPersonaScreen>
                         left: appBarHeight * 0.5,
                       ),
                       child: Image(
+                        height: 215*scaleFactor,
```

Comment:
Where does the value 215 comes from?

Reply:
I'm not sure this had the expect impact.
On a small screen, the personas are still overflowing:
![Galaxy_Nexus](https://user-images.githubusercontent.com/7460169/154864041-24b8f5f5-0d8f-46de-a4e2-0d4f021b8562.png)

On big screens, the personas are not scaling up:
![Nexus_10](https://user-images.githubusercontent.com/7460169/154864042-00408b3e-147c-4286-9e53-3e9d9e15858c.png)